.buildkite/nightly-benchmarks/README.md

```diff
@@ -31,16 +31,30 @@ Performance benchmark will be triggered when:
 - A PR being merged into vllm.
 - Every commit for those PRs with `perf-benchmarks` label AND `ready` label.
 
+Manually Trigger the benchmark
+
+```bash
+bash .buildkite/nightly-benchmarks/scripts/run-performance-benchmarks.sh
+```
+
+Runtime environment variables:
+- `ON_CPU`: set the value to '1' on Intel® Xeon® Processors. Default value is 0.
+- `SERVING_JSON`: assign a json file instead of the default one for serving tests. Default value is empty string.
+- `LATENCY_JSON`: assign a json file instead of the default one for latency tests. Default value is empty string.
+- `THROUGHPUT_JSON`: assign a json file instead of the default one for throughout tests. Default value is empty string.
+- `REMOTE_HOST`: set the remote vLLM IP to benchmark on a remote vLLM service. Default value is empty string.
+- `REMOTE_PORT`: set the remote vLLM Port to benchmark on a remote vLLM service. Default value is empty string.
+
 Nightly benchmark will be triggered when:
 - Every commit for those PRs with `perf-benchmarks` label and `nightly-benchmarks` label.
 
 ## Performance benchmark details
 
-See [performance-benchmarks-descriptions.md](performance-benchmarks-descriptions.md) for detailed descriptions, and use `tests/latency-tests.json`, `tests/throughput-tests.json`, `tests/serving-tests.json` to configure the test cases.
-
+See [performance-benchmarks-descriptions.md](performance-benchmarks-descriptions.md) for detailed descriptions, and use `tests/latency-tests-gpu.json`, `tests/throughput-tests-gpu.json`, `tests/serving-tests-gpu.json` to configure the test cases.
```

Comment:
I also think adding ```-gpu``` suffix is not necessary as it will introduce lots of changes.

Reply:
changed it accordingly.